sdk/textanalytics/cognitiveservices-textanalytics/README.md

```diff
@@ -1,6 +1,13 @@
 # Azure TextAnalytics client library for JS
 
-[Azure TextAnalytics](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes four main functions: sentiment analysis, key phrase extraction, language detection, and entity recognition.
+[Azure TextAnalytics](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
+
+- Language Detection
+- Sentiment Analysis
+- Key Phrase Extraction
+- Named Entity Recognition
+- Recognition of Personally Identifiable Information
+- Linked Entity Recognition
```

Comment:
The text right below this bullet list has one liners for each of the points above.
Any chance of consolidating the two?

Reply:
The two lists are very similar, but I think of the below as more of a list of use cases, which consolidates the three entity recognition cases into one example sentence.

Reply:
I guess I'm not sure if we need to have exactly as many example sentences as the list of function endpoints

Reply:
(Also I stole these from the official MSDN service docs)